docs/local-development.md

```diff
@@ -0,0 +1,157 @@
+# Local Development Setup Guide
+
+This guide will help you set up a local development environment for the Restarters.net project using Docker.
+
+- [Local Development Setup Guide](#local-development-setup-guide)
+  - [Prerequisites](#prerequisites)
+  - [Setup Steps](#setup-steps)
+    - [1. Clone the Repository](#1-clone-the-repository)
+    - [2. Environment Configuration](#2-environment-configuration)
+    - [3. Starting the Development Environment](#3-starting-the-development-environment)
+    - [4. Initial Setup](#4-initial-setup)
+    - [5. Accessing the Application](#5-accessing-the-application)
+  - [Common Tasks](#common-tasks)
+    - [Running Commands in the Core Application Container](#running-commands-in-the-core-application-container)
+    - [Stopping the Environment](#stopping-the-environment)
+    - [Rebuilding Containers](#rebuilding-containers)
+****
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- [Docker](https://docs.docker.com/get-docker/) (newer versions already include Docker Compose)
+  - [Docker Compose](https://docs.docker.com/compose/install/) (only needed for older Docker versions)
+- [Task](https://taskfile.dev/installation/)
+
+> [!NOTE]
+> The `Taskfile` will automatically detect your Docker version and use the appropriate command (`docker-compose` or `docker compose`).
+
+## Setup Steps
+
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/TheRestartProject/restarters.net.git
+cd restarters.net
+```
+
+### 2. Environment Configuration
+
+Copy the example environment file:
+
+```bash
+cp .env.example .env
+```
+
+You may need to adjust the following settings in your `.env` file:
+- The `APP_KEY` will be auto-generated during setup
+> [!NOTE]
+> This is only required if another service is already running on the port `8001`
+- Set `APP_URL` to `http://localhost:<AVAILABLE_PORT>`
```

Comment:
I think you only need to do this if the default (8001?) is not available.  Better to word it like that.

Reply:
```diff
 - The `APP_KEY` will be auto-generated during setup
 > [!NOTE]
-> This is only required if another service is already running on the port `8001`
-- Set `APP_URL` to `http://localhost:<AVAILABLE_PORT>`
+> This is only required if the default port, `8001`, is not available.
+- Set `APP_URL` to `http://localhost:PORT
```

Is this what you had in mind?

Reply:
Yup.